### ISSUE TITLE
Mesh cannot be decoded after sequential encoding

### ISSUE THREAD

**Issue by donmccurdy** [google/draco#1013]
I have a user-reported error related to a file that becomes invalid (the decoder always fails) after Draco encoding with the sequential method. I've stripped everything unrelated from the file, so it contains a only single mesh primitive in a basic scene:

**File:** [PrimSimple.zip](https://github.com/google/draco/files/11996118/PrimSimple.zip)

Draco compression in glTF Transform or glTF Pipeline gives the same result – edgebreaker compression yields a valid file, and sequential compression yields a broken one. Files were tested in multiple engines, and none could decode the broken version. 

The original file includes morph targets, so while edgebreaker is a workaround in this simplified case, it's not an option for the source asset. Related: https://github.com/google/draco/issues/929#issuecomment-1584913857.

Example with glTF Transform CLI (https://gltf-transform.dev/cli) –

```bash
gltf-transform draco PrimSimple.gltf PrimSimple_draco_seq.glb --method sequential
gltf-transform inspect PrimSimple_draco_seq.glb 
# error!
```

**Comment by ondys** [comment]
this will be fixed in the next release.

**Comment by hybridherbst** [comment]
@ondys is this fixed in a current release already? If so, which?

**Comment by ondys** [comment]
Yes this should be fixed in the 1.5.7 release
